src/forms/renderers/MultiLineSecret.tsx

```diff
@@ -47,9 +49,6 @@ const MultiLineSecretRenderer = (props: any) => {
         [handleChange, path]
     );
 
-    // Make it a password again so the input is masked
-    uischema.options.format = 'password';
-
```

Comment:
This is what caused the secret button to disappear. After a single render altering the uischema breaks it forever. This was a very flawed approach to this problem.